lectures/markov_chains_I.md

```diff
@@ -610,7 +624,7 @@ $$
 ```{index} single: Markov Chains; Future Probabilities
 ```
 
-Recall the stochastic matrix $P$ for recession and growth {ref}`considered above <mc_eg2>`.
+Recall the stochastic matrix $P$ for recession and growth considered in {ref}`Example 1: Economic States <mc_eg2>`.
```

Comment:
@Jiarui-ZH can you update these links to use

https://sphinx-proof.readthedocs.io/en/latest/syntax.html#referencing-examples